ingestion/src/metadata/ingestion/source/database/mysql/utils.py

```diff
@@ -97,6 +98,9 @@ def parse_column(self, line, state):
         if issubclass(col_type, SET) and "" in type_args:
             type_kw["retrieve_as_bitwise"] = True
 
+    if issubclass(col_type, (JSON)):
+        type_kw.pop("collate")
```

Comment:
hmm why are we putting "collate" here?

Reply:
Collate is keyword in Mysql but when we are passing json  as datatype at that time it is not accepting it , that's why I am  removing it from `type_kw`.